src/chains/ethereum/ethereum/tests/api/eth/sendTransaction.test.ts

```diff
@@ -47,24 +47,58 @@ describe("api", () => {
       });
 
       describe("insufficient funds", () => {
-        it("returns an error when account has insufficient funds to send the transaction", async () => {
-          const p = await getProvider({
+        const types = ["0x0", "0x1", "0x2"] as const;
+        it("returns an error when account has insufficient funds to transfer the value", async () => {
```

Comment:
I feel like we should update the "it" text for this and the next test to differentiate the two - one is rejected by the tx pool, the other by the VM

Reply:
changed!